Drift/Private/Auth/DriftUuidAuthProvider.h

```diff
@@ -12,13 +12,13 @@ class FDriftUuidAuthProvider : public IDriftAuthProvider
 public:
     FDriftUuidAuthProvider(int32 instanceIndex, TUniquePtr<IDriftCredentialsFactory> credentialsFactory, TSharedPtr<ISecureStorage> secureStorage);
 
-    FString GetProviderName() const override { return TEXT("uuid"); }
-    void InitCredentials(InitCredentialsCallback callback) override;
-    void GetFriends(GetFriendsCallback callback) override;
-    void GetAvatarUrl(GetAvatarUrlCallback callback) override;
-    void FillProviderDetails(DetailsAppender appender) const override;
+    virtual FString GetProviderName() const override { return TEXT("uuid"); }
```

Comment:
Please don't add virtual to overridden methods, override already says everything. It's an annoying habit in UE4.

Reply:
Most IDEs will flag it as not needed too.

Reply:
You're right, this is totally redundant now... (in my defense, this used to be a 'lint-clean' requirement last time I was cpp'ing).  Imma updating code analysis setting in my IDE and removing these changes...